catalog/plugin/catalog-plugin-metacard-validation/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -61,7 +61,7 @@
 
 
     <!-- Register in the OSGi Service Registry -->
-    <service ref="pre-ingest-plugin" interface="ddf.catalog.plugin.PreIngestPlugin"/>
+    <service ref="pre-ingest-plugin" interface="ddf.catalog.plugin.PreIngestPlugin" ranking="-1"/>
```

Comment:
Talked with Keith about this. We wanted the `MetacardValidityMarkerPlugin` to be the last pre-ingest plugin to process just in case any pre-ingest plugins modify/add attributes to the metacard. In fact, part of the reason for this PR was because the GeoCoder plugin was added as a pre-ingest plugin that adds the `location.country-code` attribute to the metacard but this was happening after validation which defeated the purpose.